garden/comparison.md

```diff
@@ -22,17 +22,19 @@ Force-torque | ✓ | ✓
 GPS / NavSat | ✓ |  ✓
 GPU Ray | ✓ | ✓ Renamed to GPU Lidar
 IMU | ✓ | ✓
+Logical audio sensor | ✕ | ✓
 Logical camera | ✓ | ✓
-Magnetometer | ✓ | ✓
+Magnetometer | ✕ | ✓
```

Comment:
From looking at http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1sensors_1_1MagnetometerSensor.html, I thought Gazebo-classic had magnetometer sensors. Is that not the case?

Reply:
oh, good catch. I was only looking into the plugins/ subdir and it is, suprise, in the sensors subdir :facepalm: